### PR TITLE
HWKAPM-668 Replace Externalizable with Serializable

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay could you review please?

File: api/src/main/java/org/hawkular/apm/api/model/Property.java [hunk 1]
```diff
@@ -31,7 +26,9 @@
  *
  * @author gbrown
  */
-public class Property implements Externalizable {
+public class Property implements Serializable {
+
+    private static final long serialVersionUID = -7906722328521533091L;
```
Comment (pavolloffay) [inline api/src/main/java/org/hawkular/apm/api/model/Property.java]: Do we need to use explicit `serialVersionUID` ?

Comment (pavolloffay) [inline api/src/main/java/org/hawkular/apm/api/model/Property.java]: If yes please add this generated field to all serializable classes.

Comment (objectiser) [inline api/src/main/java/org/hawkular/apm/api/model/Property.java]: Not particularly - so have updated.

**Comment by pavolloffay** [comment]
Could you please add `serialVersionUID` to the other classes which implement `Serializable` to keep consistent.

**Comment by objectiser** [comment]
@pavolloffay Thanks!

**Pull request merged**